action.yml

```diff
@@ -15,6 +15,6 @@ inputs:
 
 runs:
   using: 'docker'
-  image: 'Dockerfile'
+  image: 'docker://ghcr.io/laminas/automatic-releases:1'
```

Comment:
This is the magic: the action now uses an image on ghcr.io, which means only a pull operation is required, and no build operations!